universal-login-react/src/ui/Onboarding/Onboarding.tsx

```diff
@@ -42,8 +42,9 @@ export const Onboarding = (props: OnboardingProps) => {
                   <div className="perspective">
                     <WalletSelector
                       sdk={props.walletService.sdk}
-                      onCreateClick={async (ensName) => {
-                        await props.walletService.createWallet(ensName);
+                      onCreateClick={async (givenEnsName) => {
+                        setEnsName(givenEnsName);
```

Comment:
Set ens name for Onboarding state via state in ReactRouter's Location (like in onConnectClick callback)